src/SurveyService/Startup.cs

```diff
@@ -0,0 +1,206 @@
+using MassTransit.RabbitMqTransport;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace LT.DigitalOffice.SurveyService
+{
+  public class Startup : BaseApiInfo
+  {
+    private readonly BaseServiceInfoConfig _serviceInfoConfig;
+
+    private readonly RabbitMqConfig _rabbitMqConfig;
+
+    public const string CorsPolicyName = "LtDoCorsPolicy";
+
+
+    public IConfiguration Configuration { get; }
+
+    #region private methods
+
+    private static NewtonsoftJsonPatchInputFormatter GetJsonPatchInputFormatter()
+    {
+      var builder = new ServiceCollection()
+        .AddLogging()
+        .AddMvc()
+        .AddNewtonsoftJson()
+        .Services.BuildServiceProvider();
+
+      return builder
+        .GetRequiredService<IOptions<MvcOptions>>()
+        .Value
+        .InputFormatters
+        .OfType<NewtonsoftJsonPatchInputFormatter>()
+        .First();
+    }
+
+
+
+    #region configure masstransit
+
+    private void ConfigureMassTransit(IServiceCollection services)
+    {
+      (string username, string password) = RabbitMqCredentialsHelper
+        .Get(_rabbitMqConfig, _serviceInfoConfig);
+
+      services.AddMassTransit(busConfigurator =>
+      {
+        busConfigurator.UsingRabbitMq((context, cfg) =>
+          {
+            cfg.Host(_rabbitMqConfig.Host, "/", host =>
+              {
+                host.Username(username);
+                host.Password(password);
+              });
+
+            ConfigureEndpoints(context, cfg, _rabbitMqConfig);
+          });
+
+        ConfigureConsumers(busConfigurator);
+
+        busConfigurator.AddRequestClients(_rabbitMqConfig);
+      });
+
+      services.AddMassTransitHostedService();
+    }
+
+    private void ConfigureConsumers(IServiceCollectionBusConfigurator x)
+    {
+    }
+
+    private void ConfigureEndpoints(
+        IBusRegistrationContext context,
+        IRabbitMqBusFactoryConfigurator cfg,
+        RabbitMqConfig rabbitMqConfig)
+    {
+    }
+
+    #endregion
+
+    #endregion
+
+    public Startup(IConfiguration configuration)
+    {
+      Configuration = configuration;
+
+      _serviceInfoConfig = Configuration
+        .GetSection(BaseServiceInfoConfig.SectionName)
+        .Get<BaseServiceInfoConfig>();
+
+      _rabbitMqConfig = Configuration
+        .GetSection(BaseRabbitMqConfig.SectionName)
+        .Get<RabbitMqConfig>();
+
+      Version = "1.0.0.0";
+      Description = "SurveyService is an API that intended to work with surveies.";
```

Comment:
May be surveys, instead of surveies? https://englishmore.ru/dictionary/mnozhestvennoe-chislo-survey

Reply:
done